transport/src/test/java/io/netty/channel/AbstractChannelTest.java

```diff
@@ -87,6 +90,25 @@ public void ensureDefaultChannelId() {
         assertTrue(channelId instanceof DefaultChannelId);
     }
 
+    @Test
+    void processIdWithProcessHandleJava9() {
+        Assumptions.assumeTrue(PlatformDependent.javaVersion() >= 9);
+        ClassLoader loader = PlatformDependent.getClassLoader(DefaultChannelId.class);
+        int processHandlePid = DefaultChannelId.processHandlePid(loader);
+        assertTrue(processHandlePid != -1);
+        assertEquals(DefaultChannelId.jmxPid(loader), processHandlePid);
+        assertEquals(DefaultChannelId.defaultProcessId(), processHandlePid);
+    }
+
+    @Test
+    void processIdWithJmxPrejava9() {
+        Assumptions.assumeTrue(PlatformDependent.javaVersion() < 9);
```

Comment:
nit: You can use `@EnabledForJreRange` and so get rid of the Assumptions.

Reply:
Unlike assumptions, `@EnabledForJreRange` is not used in netty's codebase. In particular above is used verbatim in many ssl related tests. Which one do you prefer?